app/routes.js

```diff
@@ -73,30 +74,65 @@ export const routes = (authProvider) => {
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
     const response = await postAddUser({ organisationId, data: req.body, accessToken });
     if (response.success) {
-      return res.redirect(`/organisations/${organisationId}/adduser/confirmation?userAdded=${response.userAdded}`);
+      req.userAdded = response.userAdded;
+      res.userAdded = response.userAdded;
```

Comment:
Why do we need the above 2 lines?

Reply:
se we can send the user on a query string (built in the routes)

Reply:
don't know why we need it twice though 😄

Reply:
also this is the part that is going to change to the userId when we get that returned from API

Reply:
cancel that - I was seeing if there was a way to pass data through the redirect but there isn't so I'll remove